system/doc/reference_manual/data_types.xml

```diff
@@ -87,6 +87,37 @@
 11> <input>1_234.333_333</input>
 1234.333333
 </pre>
+<marker id="numeric_comparisons" />
+    <section>
+      <title>Comparisons</title>
+      <p>Both integers and floats share the same linear order. That is,
+        <c>1</c> compares less than <c>2.4</c>, <c>3</c> compares greater than
+        <c>2.99999</c>, and <c>5</c> is equal to <c>5.0</c>.</p>
+      <p>When wanting compare an integer with another integer or a float with
```

Comment:
```suggestion
      <p>When wanting to compare an integer with another integer or a float with
```
:sweat_smile::sweat_smile::sweat_smile:

Reply:
On another note, in my previous comment I suggested "match", but you used "compare", which is certainly the better choice here.
But since you're subsequently talking only about testing for (in-)equality, it may be good to end the sentence with something like "... to check for equality or inequality", or write it like "... to use pattern matching, or the term equivalence operators <c>=:=</c> to test for equality and <c>=/=</c> to test for inequality"?

Reply:
I fear that'll just make things too wordy, it's pretty bad as it is. :thinking:

Reply:
Yeah, it is pretty hard to be concise _and_ precise :sweat: I guess the message gets across even without the equality/inequality part, so it's fine as far as I'm concerned.